integration/src/test/java/com/aws/greengrass/mqtt/moquette/ClientDeviceAuthorizerTest.java

```diff
@@ -232,4 +250,53 @@ authorizer.new ConnectionTerminationListener()
         verify(mockClientDevicesAuthService).closeClientDeviceAuthSession(DEFAULT_SESSION);
         assertThat(authorizer.getSessionForClient(DEFAULT_CLIENT, DEFAULT_PEER_CERT), nullValue());
     }
+
+    @Test
+    void GIVEN_authorizedClientAndInvalidSession_WHEN_checkValid_THEN_returnsTrue(ExtensionContext context) throws
```

Comment:
do we need this test? looks like the exact case is covered in the one below

Reply:
Right, I'll delete this test as the one following it tests the same behavior